modules/party/Functions/CheckEndDate.php

```diff
@@ -7,7 +7,7 @@ function CheckEndDate($enddate): bool|string
 {
     global $func;
 
-    if ($func->str2time($enddate) < $func->str2time($_POST['startdate'])) {
+    if ($func->str2time($enddate) > $func->str2time($_POST['startdate'])) {
         return t('Der Endzeitpunkt muss nach dem Startzeitpunkt liegen');
```

Comment:
The message `Der Endzeitpunkt muss nach dem Startzeitpunkt liegen` indicates an error, that `$enddate` is earlier than start date.
If we switch it, isn't this the wrong way?